test/box/errinj.result

```diff
@@ -415,6 +415,10 @@ s_withdata:drop()
 ---
 - error: Failed to write to disk
 ...
+-- FIXME? The space is recreated, so the s_withdata object is not valid anymore.
+s_withdata = box.space.withdata
```

Comment:
I think that technically this should be possible to fix some day (not now). Let's file a ticket for this issue and reference it here: `FIXME(gh-XXXX): The space is recreated...`.

Reply:
Done. The ticket is there already: #6470.

Reply:
Created and linked to the #9120.